src/app/directives/documentation.tpl.html

```diff
@@ -181,9 +181,17 @@ <h4 class="raml-console-resource-body-heading">
             <span ng-click="changeType($event, key, code)" ng-class="{ 'raml-console-is-active': $first}" class="raml-console-flag" ng-repeat="(key, value) in methodInfo.responses[code].body">{{key}}</span>
           </h4>
 
-          <div ng-if="responseInfo[code][responseInfo[code].currentType].example">
-            <span>Example:</span>
-            <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[code][responseInfo[code].currentType].example)"></code></pre>
+          <div ng-if="examples">
```

Comment:
I don't think so, if example and examples are undefined, $scope.examples will be also undefined. I can change the transformExample to return an empty array when example and examples are undefined